runtime_kanto/src/runtime/runtime.py

```diff
@@ -241,7 +241,7 @@ def start_kanto(spinner: Yaspin, log_output: TextIOWrapper):
 
     spinner.text = "Kanto is ready to use!"
     spinner.ok("✅")
-    kanto.wait()
+    # kanto.wait()
```

Comment:
Why do we need this wait? It suggests that one can do something here (e.g. pressing Ctrl+C). But that's not true - starting up has been finished - stopping needs to be done by "runtime down" ...
Or am I missing something?

Reply:
I guess this wait is not needed since the kanto start process is a long running task, so the process will not terminate inside that function. And as you already stated out the stopping is done with runtime down

Reply:
It was possible to stop Kanto with ctrl+c and via "runtime down", but since VS Code changed the behavior only the task is still working. So I agree that it can be removed

Reply:
Removed